public/app/features/dashboard/components/ShareModal/ShareSnapshot.tsx

```diff
@@ -80,11 +80,15 @@ export class ShareSnapshot extends PureComponent<Props, State> {
       this.dashboard.snapshot.originalUrl = window.location.href;
     }
 
+    // @ts-ignore
+    window.forceRefresh = true;
     this.setState({ isLoading: true });
-    this.dashboard.startRefresh();
+    this.dashboard.startRefresh(true);
 
     setTimeout(() => {
       this.saveSnapshot(this.dashboard, external);
+      // @ts-ignore
+      window.forceRefresh = false;
```

Comment:
We can do as well w/o this attribute, as we saw in https://github.com/percona-platform/grafana/pull/137/files

Reply:
Done 👍